src/core/components/Popover/InfoPopover/InfoPopover.stories.tsx

```diff
@@ -0,0 +1,53 @@
+import { Meta } from '@storybook/react';
+
+import IconButton from '@/core/components/Button/IconButton';
+import Icon from '@/core/components/Icon';
+import InfoPopover from '@/core/components/Popover/InfoPopover/index';
+
+const meta = {
+  title: 'core/InfoPopover',
```

Comment:
p3: 디렉터리 구조로 맞춰주는건 어떨까요?
```suggestion
  title: 'core/Popover/InfoPopover',
```